packages/@aws-cdk/aws-ecs/lib/container-image.ts

```diff
@@ -0,0 +1,40 @@
+import { ContainerDefinition } from './container-definition';
+
+/**
+ * Base class for container images
+ */
+export abstract class ContainerImage {
```

Comment:
A good TODO here would be support for private container images with a SecretsManager secret for the credentials